config/application.rb

```diff
@@ -39,7 +36,7 @@ class Application < Rails::Application
                       ManageIQ::Loggers::Base.new(Rails.root.join("log", "#{Rails.env}.log"))
                     end
 
-    config.middleware.use(Prometheus::Middleware::Collector)
-    config.middleware.use(Prometheus::Middleware::Exporter)
+    require 'manageiq-api-common'
```

Comment:
Do you need this require?  Also, any reason to be marked as WIP?

Reply:
Nope, and maybe.

I want to update common to accept a metrics prefix. Right now all the metrics are called `http_server_<metric>` *for all of the apps* So I think it would be better to update that first then allow this merge to deploy the change.